src/main/java/blackjack/view/OutputView.java

```diff
@@ -27,16 +27,20 @@ public static void printHandsWithScore(Dealer dealer, List<Player> players) {
     }
 
     public static void printResult(Result result, Dealer dealer) {
-        Map<Player, ResultStatus> playerResults = result.getResults();
-        Map<ResultStatus, Long> dealerResult = result.calculateDealerResult();
+        Map<Player, Integer> playerResults = result.getResults();
+        int dealerEarning = result.calculateDealerEarning();
 
-        System.out.println("## 최종 승패");
-        System.out.println(dealerResultMessageBuilder(dealer, dealerResult));
+        System.out.println("## 최종 수익");
+        System.out.println(dealerEarningMessageBuilder(dealer, dealerEarning));
         System.out.print(playerResultsMessageBuilder(playerResults));
     }
 
-    public static void printBust() {
-        System.out.println("버스트 되었습니다.");
+    public static void printPlayerBlackjack(String playerName) {
+        System.out.printf("%s는 블랙잭 입니다.%n", playerName);
+    }
+
+    public static void printBust(String playerName) {
+        System.out.printf("%s는 버스트 입니다.", playerName);
```

Comment:
```suggestion
        System.out.printf("%s는 버스트 입니다.%n", playerName);
```
<img width="575" alt="Screenshot 2024-03-19 at 8 12 23 PM" src="https://github.com/woowacourse/java-blackjack/assets/48251668/bb62ffd0-d8ec-4a96-837f-2a1dfd776e93">

버스트 상황 출력 뒤에 줄바꿈을 해주는건 어떨까요?
사용자가 읽기 더 편할 것 같네요

Reply:
리팩토링 후 e2e? 테스트가 부족했네요..